helm/cluster-operator-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: cluster-operator
+  namespace: giantswarm
+  labels:
+    app: cluster-operator
+spec:
+  replicas: 2
```

Comment:
Why are there two replicas of this operator?

Reply:
Good catch. I missed this. The operator should only be there ones.

Reply:
Argh. Forgot that. I already changed it to one, but from the version where I had fixed version for container image ( for local testing). After that I copied the original w/ [[ .SHA ]] which then overwrote the replica count. Just a sec, I'll fix. :)